static/js/model_manager.js

```
@@ -207,12 +210,25 @@ class DropdownManager {
         if (!this.dropdown || this.config.disabled) return;
+        
+        // 在显示当前下拉菜单前，先隐藏所有其他的下拉菜单
+        DropdownManager.hideAll();
+        
+        // 如果有 onBeforeShow 回调，先执行它
+        if (typeof this.config.onBeforeShow === 'function') {
+            await this.config.onBeforeShow();
+        }
+        
         this.updateDropdown();
         this.dropdown.style.display = 'block';
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)